docs/go-configuration.md

```diff
@@ -4,27 +4,599 @@ title: Scope Go Agent configuration
+
+    scopesql "go.undefinedlabs.com/scopeagent/instrumentation/sql"
+)
+
+func main() {
+    // ...
+
+    sql.Register("scope-postgres", scopesql.WrapDriver(&pq.Driver{}, scopesql.WithStatementValues()))
+    sql.Register("scope-mysql", scopesql.WrapDriver(&mysql.MySQLDriver{}, scopesql.WithStatementValues()))
+
+    // ...
+}
+
+// ...
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Include HTTP payloads in HTTP span tags
+
+You can include HTTP payloads in the related HTTP spans, truncated to the first 512 bytes.
+
+By default, Scope Go Agent will not send HTTP payloads as they are considered sensitive information.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Environment Variable-->
+
+```sh
+SCOPE_INSTRUMENTATION_HTTP_PAYLOADS=true
+```
+
+<!--Testing instrumentation-->
+
+```go
+import (
+    "os"
+    "testing"
+
+    "go.undefinedlabs.com/scopeagent"
+    "go.undefinedlabs.com/scopeagent/instrumentation/nethttp"
+)
+
+func TestMain(m *testing.M) {
+    // Default Http Client
+	nethttp.PatchHttpDefaultClient(nethttp.WithPayloadInstrumentation())
+
+    os.Exit(scopeagent.Run(m))
+}
+
+func TestExample(t *testing.T) {
+    // Custom http.Client instance
+    client := &http.Client{Transport: &nethttp.Transport{ PayloadInstrumentation: true }}
+
+    // ...
+}
+
+func BenchmarkExample(b *testing.B) {
+    // ...
+}
+```
+
+<!--Runtime instrumentation-->
+
+```go
+import (
+    "go.undefinedlabs.com/scopeagent/instrumentation/nethttp"
+)
+
+func main() {
+    // Default Http Client
+    nethttp.PatchHttpDefaultClient(nethttp.WithPayloadInstrumentation())
+
+    // Custom http.Client instance
+    client := &http.Client{Transport: &nethttp.Transport{ PayloadInstrumentation: true }}
+
+    // ...
+}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Changing the log folder
+
+By default, the log folder were the Scope agent stores all the execution logs depends on the platform running:
+
+```
+Linux = /var/log/scope
+Windows = ${HOME}/AppData/Roaming/scope/logs
+MacOs = ${HOME}/Library/Logs/Scope
+```
+
+> If the agent can't write to the log folder, then will ask the OS for a temp folder as: `${TMP}/scope`
+
+You can overwrite and specify a custom folder were the Scope agent will store the execution logs.
+
+```sh
+SCOPE_LOGGER_ROOT=/home/user/my-logs
+```
+
+## Disabling monkey patching
+
+To enable some auto instrumentations, the Scope agent uses monkey patching over functions of the standard `testing` package.
+
+By disabling monkey patching the following features are disabled:
+
+- `testing.T` logs instrumentation: as an alternative, you can use the log methods from the scope test struct `scopeagent.GetTest(t)` as explained in the Logging Instrumentation section.
+- Tests auto instrumentation using the `go.undefinedlabs.com/scopeagent/autoinstrument` import: as an alternative, you will have to create a `TestMain` as explained in the Installation section.
+
+To disable monkey patching:
+
+```sh
+SCOPE_DISABLE_MONKEY_PATCHING=true
+```
```

Comment:
Question: why don't we patch the testing logger directly on `autoinstrument`, so there's no need for this env var, if you don't want monkey patching, you just don't import `autoinstrument`?

Reply:
Agree, that's simpler for the user... I'm gonna do a `PR` to do that refactor.